metricshub-agent/pom.xml

```diff
@@ -277,38 +277,52 @@
 				</configuration>
 			</plugin>
 
-			<!-- assembly -->
+			<!-- maven-shade-plugin -->
 			<plugin>
-				<artifactId>maven-assembly-plugin</artifactId>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<version>3.5.2</version>
 				<executions>
 					<execution>
-						<id>metricshub-jar</id>
 						<phase>package</phase>
 						<goals>
-							<goal>single</goal>
+							<goal>shade</goal>
 						</goals>
 						<configuration>
-							<descriptors>
-								<descriptor>src/assembly/metricshub-jar.xml</descriptor>

```

Comment:
Remove `src/assembly/metricshub-jar.xml`

Reply:
Alright, I got it 😄